rmgpy/molecule/molecule.py

```diff
@@ -44,20 +44,17 @@
 from collections import OrderedDict
 import itertools
 
-import element as elements
-try:
-    import openbabel
-except:
-    pass
 from .graph import Vertex, Edge, Graph, getVertexConnectivityValue
 import rmgpy.molecule.group as gr
 from .atomtype import AtomType, atomTypes, getAtomType, AtomTypeError
 import rmgpy.constants as constants
-import rmgpy.molecule.parser as parser
-import rmgpy.molecule.generator as generator
+import rmgpy.molecule.element as elements
+import rmgpy.molecule.converter as converter
+import rmgpy.molecule.translator as translator
```

Comment:
since converter and translator import Atom and Bond, would moving these import statements within the Molecule class avoid circular imports, or is this change just a placebo?

Reply:
You're right. This doesn't actually eliminate these circular imports. These changes were originally motivated by me running into circular import issues when trying to add new functionality, but I've forgotten the exact situation.